OKTA-527704 - Add note for ENG_IDX_ENABLE_DEFAULT_MFA_POLICY_IN_AUTHENTICATOR_FORMAT (#3599)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/policy/index.md

> **Note:** The `authenticators` parameter allows you to configure all available authenticators, including authentication and recovery. In contrast, the `factors` parameter only allows you to configure multifactor authentication.

> > **Note:** For orgs with the Authenticator enrollment policy feature enabled, the new default authenticator enrollment policy created by Okta contains the `authenticators` property in the policy settings. Existing default authenticator enrollment policies from a migrated Classic Engine org remains unchanged and still uses the `factors` property in its policy settings.

Reviewer: ```suggestion
> **Note:** For orgs with the Authenticator enrollment policy feature enabled, the new default authenticator enrollment policy created by Okta contains the `authenticators` property in the policy settings. Existing default authenticator enrollment policies from a migrated Classic Engine org remain unchanged and still use the `factors` property in their policy settings.
```
